Northstar.CustomServers/mod/scripts/vscripts/titan/_replacement_titans_drop.gnut

```diff
@@ -165,6 +165,11 @@ Point function CalculateTitanReplacementPoint( vector playerOrg, vector playerEy
 					Point point
 					point.origin = dropPoint
 					point.angles = yawAngles
+					
```

Comment:
dont add gamemode specific stuff to these files make it a callback and set it in the gamemode file